docs/pages/admin-guides/teleport-policy/integrations/netiq.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Discover NetIQ Access Patterns with Teleport Policy
+description: Describes how to synchronize OpenTex NetIQ access patterns using Teleport Policy and Access Graph.
+---
+
+With Teleport Policy's Access Graph, you gain insights into your NetIQ organization
+structure, the resources each user can access, and the roles that grant them access.
+Access Graph provides a visual representation that enhances security and improves
+understanding of large NetIQ organizations.  
+
+Access Graph helps answer key questions such as:  
+
+- **Which resources does each user have access to?**  
+- **Which roles grant access to specific resources?**  
+
+Access Graph is a feature of the [Teleport Policy](https://goteleport.com/platform/policy/) product, available to **Teleport Enterprise edition** customers.  
+
+If enabled, Teleport Policy options can be found under the **Policy** section in the left navigation menu.  
+
+## How it works
+
+Access Graph synchronizes various NetIQ resources, including users, resources, roles, and groups.
+These resources are then visualized in a graph representation, detailed in the
+[Access Graph page](../teleport-policy.mdx).  
+
+The import process involves two primary steps:  
+
+### Querying NetIQ APIs
+
+The Teleport cluster continuously scans the configured NetIQ organization and retrieves the
+following resources:  
+
+- Users  
+- Groups  
+- Resources  
+- Roles (Business, Permissions, and IT Roles)  
+- Role and Group memberships  
+
+Once all necessary resources are fetched, Teleport pushes them to the Access Graph, 
+ensuring it remains updated with the latest information from your NetIQ organization.  
+
+### Importing resources
+
+Teleport Policy’s Access Graph processes the imported resources 
+and their relationships, generating a graphical representation
+to visualize access structures effectively.  
+
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v17.3.0 or later.
+- Teleport Policy enabled for your account.
+- An OpenTex NetIQ instance with a user that has read access to the organization.
+- For self-hosted clusters:
+  - Ensure that an up-to-date `license.pem` is used in the Auth Service configuration.
+  - A running Access Graph node v1.27.0 or later.
+Check the [Teleport Policy page](../teleport-policy.mdx) for details on
+how to set up Access Graph.
+  - The node running the Access Graph service must be reachable from the Teleport Auth Service.
+
+## Step 1/3. Create NetIQ IDM OAuth Client
+
+To register a new OAuth client with OSP (IDM Authorization Server), modify the 
+`ism-configuration.properties` file.  
+
+The file is located in the `{osp-path}/tomcat/conf/` directory.  
```

Comment:
Should we specify which server includes this file, or will NetIQ users know this already? (I'm new to NetIQ)

Reply:
I think they will know. We are changing the OSP component of netiq. It's difficult to give an official answer because you can deploy netiq in k8s, linux and windows. 

OSP is the component we want to change the configuration for

Reply:
Works for me!